src/scenes/ProgressReports/Detail/WorkflowCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Card, CardActions, StepperProps, Typography } from '@mui/material';
+import { ProgressReportStatus as Status } from '~/api/schema/schema.graphql';
+import { RelativeDateTime } from '~/components/Formatters';
+import { ButtonLink } from '~/components/Routing';
+import { ProgressReportDetailFragment } from './ProgressReportDetail.graphql';
+import { StatusStepper } from './StatusStepper';
+
+type WorkflowCardProps = {
+  report?: ProgressReportDetailFragment;
+} & StepperProps;
+
+export const WorkflowCard = ({ report, ...rest }: WorkflowCardProps) => {
+  const currentStatus = report?.status.value as Status | undefined;
+
+  const currentWorkflowEvent = report?.workflowEvents.at(
+    report.workflowEvents.length - 1
+  );
+  return (
+    <Card {...rest}>
+      <Typography variant="h3" sx={{ pl: 2, pt: 2 }}>
+        Status
+      </Typography>
+      <StatusStepper loading={!report} current={currentStatus} />
+      {currentWorkflowEvent && (
+        <CardActions
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+          }}
+        >
+          <ButtonLink to="workflow-history">View History</ButtonLink>
+          <Typography variant="caption" sx={{ color: 'gray', pr: 1 }}>
```

Comment:
Colors should never be hardcoded. This might not seem like you are referencing a browser preset, but it's no different than saying `#808080`.
```suggestion
          <Typography variant="body2" color="text.secondary" sx={{ pr: 1 }}>
```